src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py

```diff
@@ -69,7 +69,6 @@ def __init__(self, job_id, random_seed, area, should_export_plots):
         }
 
         self.bids_offers_trades = {}
-        self.last_energy_trades_high_resolution = {}
         self.results_handler = self._create_endpoint_buffer(should_export_plots)
         self.simulation_state = {"general": {}, "areas": {}}
 
```

Comment:
The purpose of high resolution energy trades was to create plots of energy trades(which were of high resolution)?

Reply:
I'm not sure exactly. Maybe @spyrostz can help us here?

Reply:
Exactly. The default resolution of our graphs is tied to the market slot, so with default parameters 15 minutes. These graphs intended to display graphs with lowef resolution than 15-mins, typically second resolution (IIRC in order to indicate the time that the offer / bid was posted, and that the trade was created during a market slot). However these graphs will not be needed now, especially in the gsy-e (we created these in the past in order to use them for a research project, however I doubt that we will need them in the future)